packages/flutter_tools/lib/src/android/android_device.dart

```diff
@@ -89,6 +89,12 @@ class AndroidDevice extends Device {
   final String modelID;
   final String? deviceCodeName;
 
+  @override
+  DeviceConnectionInterface get connectionInterface =>
+      id.contains('_adb-tls-connect')
```

Comment:
is there any documentation to this magic string we can link to from a comment?

Reply:
I discovered it by trying it out, but I just found this: https://android.googlesource.com/platform/packages/modules/adb/+/refs/heads/master/adb_mdns.h#30
so maybe I should take out the leading `_`. Should I add a comment linked to the android source code?